src/VASAutoBalance.cpp

```diff
@@ -207,14 +216,21 @@ class VAS_AutoBalance_PlayerScript : public PlayerScript
 
         void OnLogin(Player *Player)
         {
-            if (enabled)
+            bool enabled = Player->CustomData.GetDefault <AutoBalancePlayerInfo>("AutoBalancePlayerInfo")->enabled;
```

Comment:
Why this?

Reply:
Maybe it's because the if below, but if it is, you forgot to add in.

Reply:
1 is so each individual player can disable as it uses datamap and other is a conf one so it pertinent to enable or disable